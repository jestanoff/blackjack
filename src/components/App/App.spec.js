import React from 'react'
import App from './App'
import renderer from 'react-test-renderer'

describe.skip('<App />', () => {
  it('should render proper markup', (): void => {
    const tree = renderer.create(<App />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
