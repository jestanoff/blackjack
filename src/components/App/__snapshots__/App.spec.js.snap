// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render proper markup 1`] = `
<div
  className="app"
>
  <h1
    className="text text-color-white text-size-huge"
  >
    Lystable Blackjack
  </h1>
  <section
    className="board"
  >
    <section
      className="board--cards"
    >
      <div
        className="board--ai-cards"
      >
        <div>
          <h2>
            AI cards:
          </h2>
          <p>
            sum 
            10
          </p>
        </div>
        <div
          className="card card-suit-club card-value-J"
        />
      </div>
      <div
        className="board--player-cards"
      >
        <div>
          <h2>
            Player cards:
          </h2>
          <p>
            sum 
            12
          </p>
        </div>
        <div
          className="card card-suit-heart card-value-K"
        />
        <div
          className="card card-suit-club card-value-2"
        />
      </div>
    </section>
    <section
      className="board--controls"
    >
      <h1
        className="text text-color-dark-grey text-size-standard"
      >
        Players turn
      </h1>
      <hr
        className="board--controls-hr"
      />
      <button
        className="button button-color-tertiary"
        onClick={[Function]}
      >
        Hit
      </button>
      <button
        className="button button-color-primary"
        onClick={undefined}
      >
        Stick
      </button>
      <hr
        className="board--controls-hr"
      />
      <button
        className="button button-color-warning"
        onClick={[Function]}
      >
        Reset
      </button>
    </section>
  </section>
</div>
`;
